pynest/examples/sudoku/sudoku_solver.py

```diff
@@ -122,9 +123,10 @@
 
 img_name = "sudoku_solution.png"
 logging.info(f"storing final state to: {img_name}...")
-out_image = plot_field(puzzle, solution, True)
-out_image.show()
-out_image.save(img_name)
+fig, ax = plt.subplots(1, 1)
```

Comment:
One column and one row is the default.
```suggestion
fig, ax = plt.subplots()
```